src/firebase.ts

```diff
@@ -0,0 +1,29 @@
+import { initializeApp } from 'firebase/app'
+import { getMessaging } from 'firebase/messaging'
+import { getId, getInstallations } from 'firebase/installations'
+
+const firebaseConfig = {
+  apiKey: 'AIzaSyDgtB_hqwL1SS_YMYepRMmXYhmc7154wmU',
+  authDomain: 'adamant-messenger.firebaseapp.com',
+  databaseURL: 'https://adamant-messenger.firebaseio.com',
+  projectId: 'adamant-messenger',
+  storageBucket: 'adamant-messenger.appspot.com',
+  messagingSenderId: '987518845753',
+  appId: '1:987518845753:web:6585b11ca36bac4c251ee8'
+}
```

Comment:
We use same config in the app and in the Service Worker. Let's move the const into another place and import it.